products/design-system-forms-library/products/components/va-file-input/design-decision-log.md

```diff
@@ -15,15 +15,15 @@ Last updated: 07/17/2025
 Teams wanting to implement the `va-file-input` web component often need support for encrypted and password protected files. However, the current implementation of `va-file-input` does not determine if a file is encrypted nor does it decrypt the file. This has been a surprise for some teams who expected an all-in-one solution.
 
 ### Decision
-The `va-file-input` web component will support setting the `encypted` parameter to `true` when the user needs to supply a password. This will display a password field. This password can be retrieved through the `vaPasswordChange` event. All additional functionality related to decrypting the file or using the provided password will need to be handled by the individual team.
+The `va-file-input` web component will support setting the `encrypted` property to `true` when the user needs to supply a password. This will display a password field. This password can be retrieved through the `vaPasswordChange` event. All additional functionality related to decrypting the file or using the provided password will need to be handled by the individual team.
 
-This decision has been made because of a philisophical approach to building web components that strives to keep the web component as "dry" as possible and to decouple business requirements from the standardized UI. Each team will have a different approach and use case for how they need to handle encrypted and password protected files and the web component should not try to create a single solution for all of those situations. Our team has performed research that revealed there are many unique ways to handle encryption, which reinforces the previous notion. Additionally, the majority of the file related logic should live server side, which is where the file will likely be stored and processed.
+This decision has been made because of a philisophical principle to build components that strives to keep the web component as "DRY" as possible and to decouple business requirements from the standardized UI. Each integration could potentially have a different approach and use case for how they need to handle encrypted and password protected files and the web component should not try to create a single solution for all of those situations. Our team performed research that revealed there are many unique ways to handle encryption. This principle also promotes modularity and allows different parts of an application to be developed, tested, and updated independently. 
```

Comment:
> Our team performed research that revealed there are many unique ways to handle encryption

@powellkerry Do have a link to this ticket by chance?

@ndsprinkle Would it make sense to link to that in this doc?

Reply:
Here is the original discovery ticket for encryption: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3601

Reply:
Yeah I think adding it is a great idea!